qutebrowser/browser/webengine/webview.py

```diff
@@ -261,3 +264,30 @@ def acceptNavigationRequest(self,
                                                  is_main_frame=is_main_frame)
         self.navigation_request.emit(navigation)
         return navigation.accepted
+
+    def chooseFiles(self, mode, old_files, accepted_mimetypes):
```

Comment:
This shouldn't be in `webview.py`, I try to keep `WebEnginePage`/`WebEngineView` relatively slim wrappers over the respective Qt classes. Instead, this should go to `shared.py`, at which point it hopefully is also easy to use with QtWebKit as well.